connector/src/yang/connector/netconf.py

```diff
@@ -383,12 +524,15 @@ def connect(self):
 
         try:
             self.session.connect(**defaults)
-            logger.info('NETCONF CONNECTED')
+            self.log.info('NETCONF CONNECTED')
         except Exception:
             if self.session.transport:
                 self.session.close()
             raise
 
+        # disable info logging for ncclient
+        nccl.setLevel(logging.WARNING)
```

Comment:
What messages are we loosing by setting this to WARNING?  Users should see close to what ncclient is seeing over the wire in order to debug device issues.

Reply:
I have updated the logging level change to only change it right before we call the low level request. The device log will contain the sent and received XML messages, by default they are formatted in pretty-print form.

Reply:
ncclient logging has changed over the years.  As I remember it, the sent RPC used to show up in INFO logs but the rpc-reply showed up in DEBUG level.  That was probably to protect user content from being added to logs unless the user changed to DEBUG level.  If the rpc-reply now shows up at INFO level, this change is fine.